airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/command/MockDestinationCatalogFactory.kt

```diff
@@ -13,21 +13,6 @@ import io.micronaut.context.annotation.Primary
 import io.micronaut.context.annotation.Requires
 import jakarta.inject.Singleton
 
-/**
```

Comment:
I had to remove this to get tests to pass, because moving this to `testFixtures` brought it into scope for integration tests, which caused it to override the file-injected catalog. Rather than mess around with properties, I just deleted the default test behavior.